www-apps/prowlarr/files/prowlarr.init

```diff
@@ -0,0 +1,41 @@
+#!/sbin/openrc-run
+
+depend() {
+	need net
+}
+
+run_dir="/run/prowlarr"
+pidfile="$run_dir/prowlarr.pid"
+etc_dir="/etc/prowlarr"
+var_dir="/var/lib/prowlarr"
+log_dir="/var/log/prowlarr"
+log_file="prowlarr.log"
+program="$APP_DIR/Prowlarr"
+
+check_config() {
+	if [ ! -d "${run_dir}" ]; then
+		mkdir "${run_dir}"
+	fi
+
+	checkpath -d -m 0750 -o "${USER}:${GROUP}" "${run_dir}"
+	checkpath -d -m 0775 -o "${USER}:${GROUP}" "${var_dir}"
+	checkpath -d -m 0750 -o "${USER}:${GROUP}" "${etc_dir}"
+}
+
+start() {
+	check_config
+
+	ebegin "Starting Prowlarr"
+
+	start-stop-daemon --start --background --make-pidfile --pidfile "${pidfile}" \
```

Comment:
For new init scripts, it's recommended not to define your own start/stop functions and instead define variables that will be used by the default start function.

See https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#dont-write-your-own-startstop-functions and openrc-run(8).

Reply:
I've greatly simplified these rc scripts, can you please give them a look over now?